### PR TITLE
Sinatra isps

### LINKED ISSUE #4: Set up utility endpoints in sinatra
* [x] Create electricity endpoint

### LINKED ISSUE #5: Set up Simplecov for sinatra

### LINKED ISSUE #9: Set up Utility Endpoints for Sinatra
* [x] Add electricity endpoint
* [x] Add other endpoints

### PR DESCRIPTION
**Pull request by BJSherman80**
Description:  Added endpoints for water internet and tv with passing spec 
* 

Fixes #

* [x] All tests passing
* [x] Lint code
* [x] Checked local host
* [x] Updated README

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ class ApplicationController < Sinatra::Base
     JSON.generate(message: "Welcome to the Yelp Microservice!")
   end
 
+
```
Comment (jakeheft) [inline app/controllers/application_controller.rb:14]: Remove blank line

File: app/serializers/utility_serializer.rb [hunk 2]
```diff
@@ -5,6 +5,7 @@ def self.make_json(data)
 		{
 			'data': data[:businesses].map do |biz|
 				{
+					'id': biz[:id],
```
Comment (jakeheft) [inline app/serializers/utility_serializer.rb]: Thanks for adding this - we'll adjust things on FE as well

File: spec/application_controller_spec.rb [hunk 3]
```diff
@@ -6,6 +6,79 @@ def app
...
+           "review_count": 84,
+           "rating": 1.5,
+           "categories": [
+               "Electricity Suppliers"
+           ],
+           "location": {
+               "address1": "1099 18th St",
+               "address2": "Ste 3000",
+               "address3": "",
+               "city": "Denver",
```
Comment (jakeheft) [inline spec/application_controller_spec.rb:25]: Should this be moved into a webmock/fixture file? Not entirely sure what best practice is...

**Review by jakeheft — APPROVED** [review]
Tentatively approved if you want to keep the hash data in application controller spec as is.

File: spec/application_controller_spec.rb (hunk 3 above)
Comment (Callbritton) [reply spec/application_controller_spec.rb:25]: Brett and I are planning to take a look at alternative testing options for this! We wanted to add a test that would confirm the structure of our data, but will need to also confirm the API call is successful without continuous API calls happening!

File: app/serializers/utility_serializer.rb (hunk 2 above)
Comment (Oxalisviolacea) [reply app/serializers/utility_serializer.rb]: Thank you!

**Review by Oxalisviolacea — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -18,17 +18,20 @@ How to run local host:
...
+### Utility Requests:
 ```
-GET '80211/utilities/electricity'
+GET ':location/utilities/electricity'
+GET ':location/utilities/water'
+GET ':location/utilities/tv'
+GET ':location/utilities/isps'
 ```
-Response
+Example Response for `GET '80211/utilities/electricity'`
```
Comment (jakeheft) [inline README.md:34]: Nice updates on the README

**Review by jakeheft — APPROVED** [review]

**Pull request merged**